desktop/core/src/desktop/settings.py

```diff
@@ -31,6 +31,8 @@
 import uuid
 
 import desktop.redaction
+from desktop.lib import conf
```

Comment:
is this and other changes being forced by ruff?

Reply:
I assume some of the changes are just linting, but why the added imports? Were they missing?

Reply:
seems from line 76 and 77, but ruff reminder this by auto detection of some rules?

Reply:
Yes this particular changes was flagged by ruff check from line 76-77 that imports need to be on top, I added them on the top. Other changes in this file were also flagged by ruff because I removed pylint reference from this file.

This also shows that the check is working fine and at par with the previous pylint way.